### ISSUE TITLE
Importing data to checkboxes

### ISSUE THREAD

**Issue by sunnylawn** [putyourlightson/craft-campaign#380]
I'm wanting to import some data onto pre-existing Contacts.

They have a checkbox, with three options available, when importing, contacts that only have one of the options available works no problem, but when importing more than on of the options it fails with a 'Festival is invalid." With festival being being the name of the field, and the options being North, South, Scotland.

I'm importing from a CSV with the following format:

`email@email.com,firstName,lastName,"north"`

The above works, but when adding in a second:

`email@email.com,firstName,lastName,"north,south"`

It fails? Is there a specific format for importing multiple options, or am I pushing it too much with that?

Thanks

**Comment by bencroker** [comment]
Can you try the following format?

```
email@email.com,firstName,lastName,"[\"north\",\"south\"]"
```

**Comment by sunnylawn** [comment]
No luck sorry, same error.

**Comment by bencroker** [comment]
It turned out that the imported value needed to be JSON decoded. I’ve added in https://github.com/putyourlightson/craft-campaign/commit/bddb1340f95cbecbe54213dbc63d19570c1ef105 and released in version 2.6.1.

Note that CSV requires that double quotes are doubled, so please test it as follows.

```
email@email.com,firstName,lastName,"[""north"",""south""]"
```

**Comment by sunnylawn** [comment]
Nailed it! Thanks Ben
